src/garage/envs/base.py

```diff
@@ -86,7 +86,12 @@ def _close_viewer_window(self):
                    for package in KNOWN_GYM_NOT_CLOSE_MJ_VIEWER):
                 # This import is not in the header to avoid a MuJoCo dependency
                 # with non-MuJoCo environments that use this base class.
-                from mujoco_py.mjviewer import MjViewer
+                try:
+                    from mujoco_py.mjviewer import MjViewer
+                except ImportError:
```

Comment:
@ryanjulian Is this what you had in mind? Or should I re-throw the exception?

Reply:
seems fair. I guess the rest is handled by gym.